docs/ref/models/options.txt

```diff
@@ -387,6 +387,28 @@ The name of the manager to use for the model's
     See :meth:`django.db.models.Model.save()` for more about the old and
     new saving algorithm.
 
+``indexes``
+-----------
+
+.. attribute:: Options.indexes
+
+.. versionadded:: 1.11
+
+    A list of :doc:`indexes </ref/models/indexes>` that you want to define on
+    the model::
+
+        from django.db import models
+
+        class FooBar(models.Model):
```

Comment:
If you could use some real model that's used elsewhere in the docs rather than a meaningless "FooBar", I think that would be an improvement.